google-cloud-firestore/lib/google/cloud/firestore/query.rb

```diff
@@ -389,7 +391,13 @@ def start_at *values
           new_query = @query.dup
           new_query ||= StructuredQuery.new
 
-          values = values.flatten.map { |value| Convert.raw_to_value value }
+          values = values.flatten.map do |value|
+            if value.respond_to? :document_id
```

Comment:
I don't know much about Ruby - but this seems like it might break if a user-specified object responds to `document_id`?

Reply:
Ruby doesn't have user-specific objects, we instead use `Hash`. If we did have user-specific objects, we would perform the conversion to `Hash` before the check for `document_id`.

Reply:
This hash been removed in the latest code change. PTAL.